components/browser/toolbar/src/main/res/layout/mozac_browser_toolbar_displaytoolbar.xml

```diff
@@ -25,16 +25,17 @@
 
     <!-- URL container -->
 
-    <ImageView
+    <androidx.constraintlayout.widget.ConstraintLayout
         android:id="@+id/mozac_browser_toolbar_background"
+        android:clipChildren="false"
```

Comment:
If you want the ripple to not be cut, this should be on the parent constraint layout.